region/maridia/inner-green/West Sand Hall.json

```diff
@@ -336,7 +336,7 @@
         "canSandfallBounce",
         "h_canUseMorphBombs",
```

Comment:
Looks like this is forcing normal Bombs, so the `canIBJ` is probably always going to be on

Reply:
I should do that.